src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/serializer_cache.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$container->loadFromExtension('framework', array(
```

Comment:
I suggest adding the "legacy" word in these fixtures so we can easily find them when preparing 4.0